README.md

```diff
@@ -18,23 +18,35 @@
-| [APM-Server](./apm-server/README.md)       | https://www.elastic.co/guide/en/apm/server/current/running-on-docker.html       |
-| [Elasticsearch](./elasticsearch/README.md) | https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html     |
-| [Filebeat](./filebeat/README.md)           | https://www.elastic.co/guide/en/beats/filebeat/current/running-on-docker.html   |
-| [Kibana](./kibana/README.md)               | https://www.elastic.co/guide/en/kibana/current/docker.html                      |
-| [Logstash](./logstash/README.md)           | https://www.elastic.co/guide/en/logstash/current/docker.html                    |
-| [Metricbeat](./metricbeat/README.md)       | https://www.elastic.co/guide/en/beats/metricbeat/current/running-on-docker.html |
+We recommend that the Helm chart version is aligned to the version of the product
+you want to deploy. This will ensure that you using a chart version that has been
+tested against the corresponding production version.  
+This will also ensure that the documentation and examples for the chart will work 
+with the version of the product you are installing.
+
+For example if you want to deploy an Elasticsearch `7.5.1` cluster, use the
+corresponding `7.5.1` [tag][elasticsearch-751].
+
+The `master` version of these charts are intended to support the latest pre-release
+versions of our products, and therefore may or may not work with current released
+versions.
+
+| Chart                                      | Docker documentation                                                            |Latest 7 Version|Latest 6 Version|
+|--------------------------------------------|---------------------------------------------------------------------------------|-----------|-----------|
+| [APM-Server](./apm-server/README.md)       | https://www.elastic.co/guide/en/apm/server/current/running-on-docker.html       |[`7.8.0`][apm-7] |[`6.8.10`][apm-6] |
+| [Elasticsearch](./elasticsearch/README.md) | https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html     |[`7.8.0`][elasticsearch-7] |[`6.8.10`][elasticsearch-6] |
+| [Filebeat](./filebeat/README.md)           | https://www.elastic.co/guide/en/beats/filebeat/current/running-on-docker.html   |[`7.8.0`][filebeat-7] |[`6.8.10`][filebeat-6] |
+| [Kibana](./kibana/README.md)               | https://www.elastic.co/guide/en/kibana/current/docker.html                      |[`7.8.0`][kibana-7] |[`6.8.10`][kibana-6] |
+| [Logstash](./logstash/README.md)           | https://www.elastic.co/guide/en/logstash/current/docker.html                    |[`7.8.0`][logstash-7] |[`6.8.10`][logstash-6] |
+| [Metricbeat](./metricbeat/README.md)       | https://www.elastic.co/guide/en/beats/metricbeat/current/running-on-docker.html |[`7.8.0`][metricbeat-7] |[`6.8.10`][metricbeat-6] |
 
 ## Supported Configurations
 
-Starting with 7.7.0 release, some charts are reaching GA.
+Starting with the `7.7.0` and `6.8.9` releases, some charts are reaching GA.
```

Comment:
```suggestion
Starting with the `7.7.0` release, some charts are reaching GA.
```
The 6.8.x releases are not going to be GA.

Reply:
OK, my mistake :)